### DISCUSSION TITLE
Difference between graphql-java / graphql-java-spring and graphql-java-kickstart / graphql-spring-boot

### DISCUSSION THREAD

**Discussion by rkagni** [graphql-java/graphql-java#2265] [Q&A]
I see lot of questions with this confusion. Which is the official version that we can rely on?

**Comment by TcPaulyue** [comment]
GraphQL Java Kickstart provide libraries for using GraphQL-Java. Some of these libraries used to be part of GraphQL-Java itself, but have been extracted.
